README.md

```diff
@@ -58,7 +58,7 @@ instructions and configurations for alternatives.
 ```bash
 # Install Ubuntu dependencies
 sudo apt update
-sudo apt install gfortran git git-lfs ninja-build cmake g++ pkg-config xxd patchelf automake libtool python3-venv python3-dev libegl1-mesa-dev
```

Comment:
A few other places we may want to update if no longer required and we don't plan on adding it back:

* https://github.com/ROCm/TheRock/blob/d85e76dbc4402c974b5410f49f95ebea84e0adb7/build_tools/fetch_sources.py#L54
* https://github.com/ROCm/TheRock/blob/d85e76dbc4402c974b5410f49f95ebea84e0adb7/build_tools/hack/env_check/check_tools.py#L244-L248

(We may want to make such changes all at once so we don't forget and leave the repo in a mixed state)

Reply:
PTAL